src/endpoints/v1.es6.js

```diff
@@ -217,14 +217,19 @@ class APIv1Endpoint {
     cache.set(key, data);
   }
 
-  get subreddits () {
+  get subreddits() {
     return bind({
-      buildOptions: function(options) {
-        var uri = this.origin + `/r/${options.query.subreddit}/about.json`;
+      buildOptions: function (options) {
+        var uri = this.origin;
+        if (options.query.where) {
+          uri += `/subreddits/${options.query.where}.json`;
```

Comment:
Can we use `sort` instead of `where`, like we do for `hot` / `best` / etc elsewhere? That parameter matches more what the query is actually doing. (`gold` is weird, but such is the API.)